Modules/PostFilter/DimmingCurve/DimmingCurveModule.cs

```diff
@@ -205,11 +205,36 @@ public ICommand Filter(ICommand command)
 				double newIntensity = _curve.GetValue(cmd.CommandValue / 255d * 100d) / 100d * Byte.MaxValue;
 				return CommandLookup8BitEvaluator.CommandLookup[(byte) newIntensity];
 			}
+			else if (command is _16BitCommand cmd16)
+			{
+				// convert command to 0 - 100 value for lookup in the curve.
+				// convert 0 - 100 return value from the curve to a percent
+				// and then mult by the max byte value to get new command value
+				double newIntensity = _curve.GetValue(cmd16.CommandValue / 65535d * 100d) / 100d * ushort.MaxValue;
+
+				return new _16BitCommand((short)newIntensity);
+			}

```

Comment:
This casting smelled when I added this a while back and now it smells twice as bad. Need to look into this at some future time to see if there is a better way.